components/Dialogue/Dialogue.tsx

```diff
@@ -56,7 +57,15 @@ const DialogActions = withStyles((theme: Theme) => ({
   },
 }))(MuiDialogActions);
 
-export default function CustomizedDialogs() {
```

Comment:
Du löschst hier eine Funktion die noch in pages/helper.tsx und pages/farm.tsx genutzt wird. Soll das so sein?